source/Grid/Grid.js

```diff
@@ -204,6 +204,8 @@ export default class Grid extends Component {
 
     // See defaultCellRangeRenderer() for more information on the usage of this cache
     this._cellCache = {}
+
+    this._scrollbarSize = getScrollbarSize()
```

Comment:
Elaborating a bit- I think the `inDom` check for `dom-helpers` will prevent scrollbar size from ever being set in that case. Because at the time the component is _created_ it won't be in the DOM.

Hmm... maybe we could address this concern by:
* Adding a fallback to `render` to handle a potentially undefined size (eg `const scrollbarSize = this._scrollbarSize || 0`)
* Adding a check in `componentDidMount` for an undefined `this._scrollbarSize` and re-initializing if necessary